### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #618 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#356]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 10.57 KB...]
lat is : 32.065163 lng is : 34.771384
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078398 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.082 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.search.SearchTest
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.505 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  AdressSearchServiceTest.searchSuccess:43

Tests run: 66, Failures: 1, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:08 min
[INFO] Finished at: 2017-05-20T12:47:38+00:00
[INFO] Final Memory: 35M/210M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Ariel Kolikant] 648ee7fa4ca3840b13e55a9c02c8b54035d87966 - adress search server side tests

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/618/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 8.98 KB...]
lat is : 32.065163 lng is : 34.771384
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078398 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.08 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.search.SearchTest
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.071 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  AdressSearchServiceTest.searchSuccess:37

Tests run: 67, Failures: 1, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:37 min
[INFO] Finished at: 2017-05-20T13:14:50+00:00
[INFO] Final Memory: 35M/207M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Ariel Kolikant] 648ee7fa4ca3840b13e55a9c02c8b54035d87966 - adress search server side tests

- [Ariel Kolikant] 284425033b5f0660225c2e5e8cb2a764c049c2ef - user information service test



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/619/)

**Comment by Kolikant** [comment]
don't worry, taking a look at it now

**Comment by JuanJenkinsBot** [comment]
Build was fixed!

**Comment by Kolikant** [comment]
hush now jenkins, the bad build can't hurt you anymore
